.nf-core.yml

```diff
@@ -0,0 +1,6 @@
+lint:
+  files_unchanged:
+    - .github/workflows/linting.yml
+    - assets/multiqc_config.yaml
+    - assets/nf-core-rnavar_logo.png
+    - docs/images/nf-core-rnavar_logo.png
```

Comment:
You should not need these two, I believe we have a regular nf-core logo for rnavar

Reply:
not sure which logo are you referring to. I haven't changed these images...

Reply:
Then, why are they here in the files_unchanged?